rclcpp/include/rclcpp/node.hpp

```diff
@@ -305,6 +311,24 @@ class Node : public std::enable_shared_from_this<Node>
   bool
   get_parameter(const std::string & name, ParameterT & parameter) const;
 
+  /// Assign the value of the map parameter if set into the values argument.
+  /**
+   * Parameter names that are part of a map are of the form "name.member".
+   * This API gets all parameters that begin with name, storing them into the
+   * map with a key of "member" and their value.  If there are no members in
+   * the named map, then the "values" argument is not changed.
+   *
+   * \param[in] name The name of the map parameter to get.
```

Comment:
From my perspective there's no such thing as a "map parameter", i.e. there's no map or dict type for parameters. I think this would be more accurately described as "the prefix of the parameters to get", or "the namespace from which to get parameters".

Reply:
Yeah, I like "prefix of the parameters to get", so I switched to that terminology.